<!--xml-->
<launch>
<node name="system_monitor" pkg="eros" type="system_monitor"  output="screen" clear_params="true">
        <param name="startup_delay"       value="0.0"/>
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="false"/>    
        <param name="loop1_rate"            value="2"/>  <!-- Update Screen -->
        <param name="loop2_rate"            value="0.2"/> <!-- Scan for new nodes.  Ping non EROS Nodes -->
        <param name="loop3_rate"            value="5"/>
</node>
</launch>
